pilot/pkg/serviceregistry/consul/conversion.go

```diff
@@ -147,3 +150,13 @@ func convertProtocol(name string) model.Protocol {
 	}
 	return protocol
 }
+
+// Extracts security option for given port from labels. If there is no such
+// annotation, or the annotation value is not recognized, returns
+// meshconfig.AuthenticationPolicy_INHERIT
+func extractAuthenticationPolicy(port int, name string) meshconfig.AuthenticationPolicy {
+	// TODO: https://github.com/istio/istio/issues/3338
+	// Check for the label - auth.istio.io/<port> and return auth policy respectively
+
+	return meshconfig.AuthenticationPolicy_INHERIT
```

Comment:
please add a testcase to conversion_test.go in this package for this case

Reply:
adding simple test without consul labels for now.